tests/lib/rules/function-call-argument-newline.js

```diff
@@ -390,6 +390,21 @@ ruleTester.run("function-call-argument-newline", rule, {
                 }
             ]
         },
+        {
+            code: "fn(a,/* commeent */\nb)",
```

Comment:
Not a blocker, but I’m assuming the intention was `comment` rather than `commeent` here 😄

Reply:
@kaicataldo Oh it's type. I fixed it. Thanks.